Makefile

```
@@ -79,6 +79,7 @@ version.h: Makefile .git/index
 	echo "#define VERSION \"$${v:-$(VERSION)}\"" >$@
 
 .git/index:
+	if [ ! -d .git ] || [ ! -f $@ ]; then mkdir .git && touch $@ ; fi
```

```suggestion
	mkdir -p .git && touch $@
```

Could it be simplified to this?